README.md

```diff
@@ -46,21 +46,14 @@ make clean # Removes the binary created above since it has already been moved
 
 After you can test to see if it works.
 
-Open a terminal and run the command:
+Run the following command and see the output and compare.
```

Comment:
```suggestion
Run the following command, see the output and compare.
```

Reply:
I don't know about this one, I'll review tomorrow